src/fidesops/api/v1/endpoints/connection_type_endpoints.py

```diff
@@ -27,41 +29,63 @@
 logger = logging.getLogger(__name__)
 
 
-def get_connection_types(search: Optional[str] = None) -> List[str]:
+def get_connection_types(
+    search: Optional[str] = None, system_type: Optional[SystemType] = None
+) -> List[ConnectionSystemTypeMap]:
     def is_match(elem: str) -> bool:
         """If a search query param was included, is it a substring of an available connector type?"""
         return search in elem if search else True
 
-    database_types: List[str] = [
-        conn_type.value
-        for conn_type in ConnectionType
-        if conn_type
-        not in [ConnectionType.saas, ConnectionType.https, ConnectionType.manual]
-        and is_match(conn_type.value)
-    ]
-    saas_types: List[str] = [
-        saas_type.value
-        for saas_type in SaaSType
-        if saas_type != SaaSType.custom and is_match(saas_type.value)
-    ]
-    connection_types: List[str] = sorted(database_types + saas_types)
+    connection_system_types: List[ConnectionSystemTypeMap] = []
+    if system_type == SystemType.database or system_type is None:
+        database_types: List[str] = sorted(
+            [
+                conn_type.value
+                for conn_type in ConnectionType
+                if conn_type
+                not in [
+                    ConnectionType.saas,
+                    ConnectionType.https,
```

Comment:
We should probably remove this value all together, since it's not used and is a duplicate of `saas`. We can do that in a follow-up.

Reply:
What do you mean it isn't used? `https` connectors are the type of connector you create if you're making a pre- or post-execution webhook.

Reply:
Ah, spoke too soon, you're talking about from the ConnectionType enum.